crates/op-rbuilder/src/builders/context.rs

```diff
@@ -399,6 +402,20 @@ impl<ExtraCtx: Debug + Default> OpPayloadBuilderCtx<ExtraCtx> {
                 }
             }
 
+            if let Some(flashblock_index) = flashblock_number {
+                if let Some(flashblock_number_min) = flashblock_number_min {
+                    if flashblock_index < flashblock_number_min {
+                        continue;
+                    }
+                }
+
+                if let Some(flashblock_number_max) = flashblock_number_max {
+                    if flashblock_index > flashblock_number_max {
+                        continue;
```

Comment:
should mark the sender as invalid as subsequent transactions from the same sender will have a nonce gap.

Reply:
would also add debug logging (I think we have a function called log_txn) that logs the reason why a transaction doesn't get included